dmci/api/worker.py

```diff
@@ -24,7 +24,7 @@
 from lxml import etree
 
 from dmci import CONFIG
-from dmci.mmd_tools import full_check
```

Comment:
can we rename `mmd_tools` to, e.g., `validator` or something?

Reply:
I changed it to just `tools`.